src/app/calculator/furnaces/cooling/liquid-cooling-form/liquid-cooling-form.component.html

```diff
@@ -0,0 +1,149 @@
+<div *ngIf="!isBaseline" class="form-group">
+  <label class="small" for="coolingMedium">Cooling Medium</label>
+    <select disabled class="form-control" id="coolingMedium" 
```

Comment:
This select menu is disabled for modification and the "coolingMedium" is set to the value of the baseline. If I select "Other Gas" in the baseline the value is "Gas" and the selected option ends up being "Gas"

Reply:
in all forms